libsolidity/codegen/ir/IRGeneratorForStatements.cpp

```diff
@@ -77,6 +77,24 @@ struct CopyTranslate: public yul::ASTCopier
 }
 
 
+string IRGeneratorForStatements::convertIfRequired(Expression const& _expression, Type const& _to)
```

Comment:
`convert_ed_IfRequired`? We don't convert the expression itself, we use a converted "copy" instead.